pydantic_extra_types/country.py

```diff
@@ -11,11 +12,11 @@
 from pydantic_core import PydanticCustomError, core_schema
 
 try:
-    import pycountry  # type: ignore[import]
-except ModuleNotFoundError:  # pragma: no cover
+    import isocountry
```

Comment:
What's this about?

Reply:
I've transitioned from using "pycountry" to the new "isocountry" library, which essentially amounts to a hard fork. You can find my customized version at https://github.com/yezz123/isocountry. It appears that "pycountry" may no longer be actively maintained, hence this shift. 😄